pkg/shell/manifests.ts

```diff
@@ -102,17 +102,27 @@ export interface Manifest {
     tools: ManifestSection | undefined;
 
     preload: string[] | undefined;
+    fullscreen: string[] | Record<string, string[]> | undefined;
     parent: ManifestParentSection | undefined;
     ".checksum": string | undefined;
 }
 
+function import_fullscreen(val: JsonValue): string[] | Record<string, string[]> {
+    if (Array.isArray(val)) {
+        return import_array(val, import_string);
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-21670-ca14cdd0-20250716-121825-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)